ui/src/Components/Global/Toggle/Toggle.js

```diff
@@ -0,0 +1,17 @@
+export default function Toggle({ onClick, active }) {
+  return (
+    <label
+      className={`w-14 h-7 rounded-3xl block relative p-1 ease-in-out duration-300 ${active ? 'bg-sky-500' : 'bg-slate-300'}`}
+    >
+      <input
+        type='checkbox'
+        className='hidden'
+        checked={active}
+        onClick={onClick}
+      />
+      <span
+        className={`w-6 h-full block rounded-3xl bg-white relative ease-in-out duration-300 ${!active && 'translate-x-full'}`}
+      />
+    </label>
+  )
+}
```

Comment:
Should it be switched to `switch`? Ref - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/switch_role